src/ck-ldb/LBManager.C

```diff
@@ -694,9 +715,24 @@ void LBManager::nextLoadbalancer(int seq)
 }
 
 // switch strategy
-void LBManager::switchLoadbalancer(int switchFrom, int switchTo)
-{
-  // TODO: Implement turn off / on
+void LBManager::switchLoadbalancer(int switchFrom, int switchTo) {
+  if(lbNames[switchTo] != "DistributedLB" && lbNames[switchTo] != "MetisLB") {
+    json config;
+    if(lbNames[switchTo] == "Hybrid") {
+      config["tree"] = "PE_Process_Root";
+      config["Root"]["pe"] = 0;
+      config["Root"]["step_freq"] = 3;
+      config["Root"]["strategies"] = {"GreedyRefine"};
+      config["Process"]["strategies"] = {"GreedyRefine"};
+    } else {
+      config["tree"] = "PE_Root";
+      config["Root"]["pe"] = 0;
+      config["Root"]["strategies"] = {lbNames[switchTo]};
+    }
+    configureTreeLB(config);
+  } else {
+    //TODO: Implement turn off / on for Distributed
```

Comment:
Is this in the pipeline of LB patches or should an issue be made for it?

Reply:
#2818